x-pack/plugins/security_solution/public/detections/components/rules/description_step/index.tsx

```diff
@@ -156,6 +159,7 @@ export const addFilterStateIfNotThere = (filters: Filter[]): Filter[] => {
   });
 };
 
+/* eslint complexity: ["error", 21]*/
```

Comment:
2️⃣ 1️⃣ ! 📈

Reply:
I was hoping you wouldn't see that one. :-) Yeah, someone needs to go in and refactor that. I kind of didn't want to add too much more additional things to this PR but we are accumulating more complexity debt and I left it as is for the next person. Probably me you know. hehe.